lib/src/disposable/disposable_vm.dart

```diff
@@ -14,22 +14,21 @@
 
```

Comment:
since this file isn't _strictly_ for the VM and instead is just platform-agnostic, could we put this in a `disposable_common.dart` file? That way it's more clear that this logic is being shared between both VM and browser